swarm/chunk/chunk.go

```diff
@@ -125,6 +138,19 @@ const (
 // ModePut enumerates different Putter modes.
 type ModePut int
 
+func (m ModePut) String() string {
+	switch m {
+	case ModePutRequest:
+		return "Request"
+	case ModePutSync:
+		return "Sync"
+	case ModePutUpload:
+		return "Upload"
+	default:
+		return "Unknown Mode Put"
```

Comment:
Same here - not sure we should keep spaces in names that might end up in metrics.

Reply:
Fixed.